File: proublemSloving_0.1/prm2.py
#Write a program that takes a list of numbers and returns the sum of all the even numbers in the list.

#For example, if the input list is [2, 5, 3, 8, 10, 12], the output should be 32 (since 2 + 8 + 10 + 12 = 32).

numbers= [2, 5, 3, 8, 10, 12]
num = 0
for i in numbers:
    if i%2 == 0 :
        num +=i
        
print (num)

File: proublemSloving_0.2/prm2.java
/*
 * Write a program in Java that takes in an array of integers and returns the second smallest number in the array. 
 * If there is no second smallest number (e.g., if all the numbers in the array are the same), the program should output an appropriate message.
 */

public class prm2 {
   

    public static void main(String[] args) {
        int[] numbers = {10, 5, 20, 15, 7, 3};
        int smallest = Integer.MAX_VALUE;
        int secondSmallest = Integer.MAX_VALUE;
    
        for (int i = 0; i < numbers.length; i++) {
            if (numbers[i] < smallest) {
                secondSmallest = smallest;
                smallest = numbers[i];
            } else if (numbers[i] < secondSmallest && numbers[i] != smallest) {
                secondSmallest = numbers[i];
            }
        }
    

        if (secondSmallest == Integer.MAX_VALUE) {
        System.out.println("There is no second smallest number.");
        } else {
        System.out.println("The second smallest number is: " + secondSmallest);
        }
    }
    
}
